<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainPage">
     <widget class="QLabel" name="dateLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#90ffc6;&quot;&gt;1 Gennaio 1970&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="humidityLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>341</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#90ffc6;&quot;&gt;Umidità: 0%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="status" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>491</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="fire">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>210</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/fire.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enablerButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Attiva</string>
      </property>
     </widget>
     <widget class="QPushButton" name="optionsButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>130</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Opzioni</string>
      </property>
     </widget>
     <widget class="QLabel" name="timeLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#90ffc6;&quot;&gt;00:00&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="tempLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>291</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:60pt; color:#90ffc6;&quot;&gt;- - -&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="optionsPage">
     <widget class="QWidget" name="titleBar" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.006, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 58, 255), stop:1 rgba(0, 2, 127, 255));</string>
      </property>
      <widget class="QLabel" name="titleLabel">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>0</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; color:#90ffc6;&quot;&gt;IMPOSTAZIONI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="Switch" name="nightSwitch" native="true">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="nightLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#90ffc6;&quot;&gt;Modalità diurna&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="Line" name="line_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="dayLimitLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#90ffc6;&quot;&gt;Soglia diurna&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="nightLimitLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#90ffc6;&quot;&gt;Soglia notturna&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dayLimitValue">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>130</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>0.00°C</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="nightLimitValue">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>210</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>0.00°C</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelButton">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-20</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/back.png);
background-repeat: repeat-n;
background-position: center;
border-width:1px;
border-radius:20px;
max-width:80px;
max-height:80px;
min-width:80px;
min-height:80px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="toolsButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>-20</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/tools.png);
background-repeat: repeat-n;
background-position: center;
border-width:1px;
border-radius:20px;
max-width:80px;
max-height:80px;
min-width:80px;
min-height:80px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pressureButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(144, 255, 198);</string>
      </property>
      <property name="text">
       <string>Pressione barometrica: -</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="toolsPage">
     <widget class="QWidget" name="toolsTitleBar" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.006, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 58, 255), stop:1 rgba(0, 2, 127, 255));</string>
      </property>
      <widget class="QLabel" name="toolsTitleLabel">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>0</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; color:#90ffc6;&quot;&gt;STRUMENTI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="poweroffButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Arresta il sistema</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="guicloserButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Chiudi l'intefaccia</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updaterButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Aggiorna il sistema</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-20</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/back.png);
background-repeat: repeat-n;
background-position: center;
border-width:1px;
border-radius:20px;
max-width:80px;
max-height:80px;
min-width:80px;
min-height:80px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="networkButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>-20</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/nonet.png);
background-repeat: repeat-n;
background-position: center;
border-width:1px;
border-radius:20px;
max-width:80px;
max-height:80px;
min-width:80px;
min-height:80px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="updatePage">
     <widget class="QLabel" name="updateLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; color:#90ffc6;&quot;&gt;Aggiornamento in corso...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="afterUpdateButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Esci</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="oskPage">
     <widget class="QLabel" name="resultLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>461</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier</family>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:50pt; color:#90ffc6;&quot;&gt;__._°C&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>461</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="spacing">
        <number>12</number>
       </property>
       <item row="2" column="0">
        <widget class="QPushButton" name="sevenButton">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="fiveButton">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="eightButton">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="fourButton">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="oneButton">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="sixButton">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="threeButton">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="twoButton">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="nineButton">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="okButton">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="cancButton">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>←</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="zeroButton">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="graphPage">
     <widget class="QWidget" name="chartView" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>461</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-20</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/back.png);
background-repeat: repeat-n;
background-position: center;
border-width:1px;
border-radius:20px;
max-width:80px;
max-height:80px;
min-width:80px;
min-height:80px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Switch</class>
   <extends>QWidget</extends>
   <header>switch.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
